.github/workflows/default_unittest.yml

```diff
@@ -0,0 +1,35 @@
+# This workflow will build a golang project
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-go
+
+name:  Unit Tests
+
+run-name: Unit tests of ${{ github.ref_type }} "${{ github.ref_name }}" by @${{ github.actor }}
```

Comment:
This information is redundant. The list of actions already has information about user and branch

Check [here](https://github.com/aergoio/aergo/actions)